django_project/certification/models/attendee.py

```diff
@@ -0,0 +1,64 @@
+# coding=utf-8
+"""
+Attendee model definitions for certification apps
+"""
+
+from django.core.urlresolvers import reverse
+from django.db import models
+from core.settings.contrib import STOP_WORDS
+from unidecode import unidecode
+from django.utils.text import slugify
+from django.contrib.auth.models import User
+
+
+class Attendee(models.Model):
+    """The person who register as attendee is defined here
```

Comment:
Please check the coding standards carefully here:

https://github.com/inasafe/inasafe/wiki/Coding-Standards#doc-strings-and-comments

so in your example it would be:

```
"""A person who is registered as an attendee for training course.

She does not attend any course unless she is defined in the 
CourseAttendee model."""
```

Note that you can just choose one gender for your sentences and use that throughout (she is fine :-) ) so that we don't have clumsy s/he everywhere.